src/common/widgets/widgetresourcedata.cpp

```diff
@@ -23,7 +23,7 @@ void InitWidgetResources(const char* filename)
 	if (!WidgetResources)
 		I_FatalError("Unable to open %s", filename);
 
-	WidgetTheme::SetTheme(std::make_unique<DarkWidgetTheme>());
+	WidgetTheme::SetTheme(std::make_unique<LightWidgetTheme>()); // GZDoom uses a different theme than VKDoom
```

Comment:
Light theme? No. It looks awful.
![image](https://github.com/user-attachments/assets/6fe5c0c1-a5b9-4cce-8290-5275ef426b34)

Reply:
Sorry if I am being too particular, this was not intended to be bikeshedding at all, and I am all for GZDoom having its own theme, but going to the light theme to accomplish that makes the launcher window look too clashy and ugly, it was too much of a step back. This was my only real complaint, so other than this, it gets merged.

Having a not 1:1 theme with VKDoom is fine and dandy, but this feels a lot more like it was an effort to make it look intentionally ugly, even if that wasn't the intent it certainly was the effect. You and I have talked at length about your misgivings with the sudden similarities with the launcher window and I agree with you, but let's take a more sane approach to this that doesn't cause 90% of the userbase to freak out at us and flip their shit.